.gitlab-ci.yml

```diff
@@ -77,29 +77,21 @@ variables:
           --tag "$IMAGE_NAME:latest"
           --file "$DOCKERFILE" .
 
-check-typing:
-  stage:                           test
-  <<:                              *common-refs
-  <<:                              *kubernetes-env
-  script:
-    - yarn --immutable --network-concurrency 1
-    - yarn typing
-
 check-linting:
   stage:                           test
   <<:                              *common-refs
   <<:                              *kubernetes-env
   script:
-    - yarn --immutable --network-concurrency 1
-    - yarn lint
+    - yarn --immutable
+    - yarn typing && yarn format && yarn lint
```

Comment:
I was confused of what this `typing` command was supposed to be. Can't it be renamed to `typecheck`? That's how it's called in other OpsTooling projects